src/app/zeko/sequencer/lib/zeko_sequencer.ml

```diff
@@ -387,6 +388,9 @@ module Make (T : Transaction_snark.S) (M : Zkapps_rollup.S) = struct
     ; snark_q : Snark_queue.t
     ; stop : unit Ivar.t
     ; da_client : Da_layer.Client.Sequencer.t
+    ; verifier : Verifier.t
```

Comment:
Did you consider copying the code?

Reply:
There isn't anything that would need a change, so no.